<?xml version="1.0"?>
<launch>

	<!-- Realsense launch -->
    <include file="$(find realsense2_camera)/launch/depth_camera_rs.launch" />

    <!-- Yolo -->
    <!-- <include file="$(find darknet_ros)/launch/yolo.launch"/> -->

    <!-- Pcl -->
<!--     <node pkg="pointcloud_processing" type="pcl_processing" name="localnavi_server" output="screen">
        <param name="DARKNET_TOPIC" value="darknet_ros/boundingbox"/>
        <param name="PCL_TOPIC" value="camera/depth/color/points"/>
        <param name="TARGET_FRAME" value="map"/>
        <param name="VISUAL" value="true"/>
        <param name="PCL_VISUAL" value="true"/>
    </node> -->
    
    <!-- TF -->
    <node name="world_to_map" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 camera_link /map 10"/>

</launch>
